contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,153 @@
+---
+date: 2024-05-28
+title: What we built at our Mykonos hackathon
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/aruba/beach-hog.jpg
+featuredImageType: full
+category: Inside PostHog
+tags:
+  - Offsites
+---
+
+Each year, the PostHog team gets together for our all-company offsite. This year, we headed to Mykonos – not to rave, but to code. And to enjoy a cocktail or two in the sun, of course.
+
+We’re a fully-remote company, with 47 misfits spread across ten countries. Our offsites are a really important part of our culture. They’re a great chance to get to know colleagues better and the connections formed at this time bring extra energy and creativity to our asynchronous work throughout the rest of the year.
+
+Our all-company offsites are a mix of socializing, group activities, strategic sessions, Post-its, workshops, more Post-its, and, the star of the show, the annual PostHog hackathon. Everyone in the company has to pitch a couple of ideas, then we all vote on our favorites, assemble teams, and have a little over a day to go from pitch to demo. And, in some cases, our hackathon projects are ready to ship right away.
+
+Here’s what we built in Mykonos.
+
+### The presidential briefing
+
+<BuiltBy people={['Charles Cook', 'Eric Duong', 'James Hawkins', 'James Temperton']} />
+
+As PostHog grows as a company, keeping track of everything that’s happening will get harder. Yes, we [write everything down](/newsletter/remote-working) but that creates a lot of reading, and a lot of noise. The solution? An AI-generated briefing, tailored to each individual team member and their interests.
+
+The Presidential Briefing was built by scraping PRs and Issues from GitHub, along with Slack messages, and then training an LLM to understand what’s interesting and important. The bot then produces a pithy briefing that removes the noise and gives people just the information they need. 
+
+In future, we’d want to add more data sources and build it using Llama to avoid the need to send any data to external companies.
+
+### MykoLogs
+
+<BuiltBy people={["Paul D'Ambra", "Tom Owers", "Ted Kaemming"]} />
+
+MykoLogs is a logging product that integrates with the existing PostHog SDKs, bringing backend logs straight into a shiny new product on PostHog. The best part? You can link backend logs to session recordings through the user’s session ID – letting you debug what was happening on the backend during your user's API requests. MykoLogs plays nicely with all other PostHog products, meaning logs and session replay are now BFFs. Debugging has never been this breezy!
+
+### Hog
```

Comment:
I think this is a really cool hackathon project which could be very important to the future of the company. Feels like we're burying the lede a little with this header. 

```suggestion
### The Hog programming language
```